src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -99,6 +99,12 @@ protected internal struct ImportModuleOptions
             /// This will be allowed when the manifest explicitly exports functions which will limit all visible module functions.
             /// </summary>
             internal bool AllowNestedModuleFunctionsToExport;
+
+            /// <summary>
+            /// Flag that controls Export-PSSession -AllowClobber parameter in generating proxy modules from remote sessions.
+            /// Historically -AllowClobber in these scenarios was set as True.
+            /// </summary>
+            internal bool NoClobberExportPSSession;
```

Comment:
It seems unusual to me that this and the field above are both fields rather than properties, but not private to this class. I think they should be changed to properties.

Reply:
The other thing is that this is a boolean field with `No` in the name, but only gets used in the negated form. That makes sense for a publicly exposed switch parameter, but I think for an internal field/property it's more confusing that simply calling it `ClobberExportPSSession`

Reply:
that would  force changing the default, and add another term for the same concept.
`NoClobber` is the most famous one, that's why I went with it.